src/main/rx/rx.c

```diff
@@ -79,6 +79,7 @@ static pt1Filter_t frameErrFilter;
 
 #ifdef USE_RX_LINK_QUALITY_INFO
 static uint16_t linkQuality = 0;
+static uint8_t rfMode = 0;
```

Comment:
This looks good so far, well done.

A couple of ideas for improvement (but they can be done in a future pull request:

- since `rfMode` is really one out of a list of discrete modes, it might make sense if we defined an enum (think of it as a codified comment) listing all the modes, and then typed `rfMode` as this;
- not sure if setting `rfMode` to 0 on signal loss is the right thing to do - unless `rfMode` 0 means 'no connection' - leaving it whatever it is is probably better, making this into 'the last mode that was active before we lost the link'.